projects/uriparser/build.sh

```diff
@@ -18,14 +18,13 @@
 # build project
 mkdir -p build
 cd build
-cmake -DCMAKE_BUILD_TYPE=Release -DURIPARSER_BUILD_DOCS:BOOL=OFF -DBUILD_SHARED_LIBS:BOOL=OFF -DURIPARSER_BUILD_TESTS:BOOL=OFF ..
+cmake -DCMAKE_BUILD_TYPE=Release \
```

Comment:
```suggestion
cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo \
```
?

Most seem to use Release here but doesn't that hurt crash report quality?

Reply:
I see that some of the projects use Release, and some use RelWithDebInfo. Not sure which approach is better. I changed to RelWithDebInfo, maybe OSS-Fuzz maintainers can provide more info here